RepClient/src/app/home/home.page.html

```diff
@@ -4,18 +4,32 @@
 <ion-content>
 	<ion-grid>
 		<ion-row id="mainContent" *ngIf="companies; else loading">
+			<!--In case companies are not undefined but empty-->
+			<ion-col size="12" *ngIf="companies.length === 0">
+				<ion-card>
+					<ion-card-header>
+						<ion-card-subtitle></ion-card-subtitle>
+						<ion-card-title>No Suppliers registered yet</ion-card-title>
+					</ion-card-header>
+					<ion-card-content>
+						<p>We couldnt find any suppliers registered with you yet pelase contact your administrator
+						to add them!</p>
+					</ion-card-content>
+					<hr>
+				</ion-card>
+			</ion-col>
 			<ion-col size="6" *ngFor="let company of companies">
 				<ion-card>
-					<img [src]="company.imageUrl" alt=""/>
+					<!--<img [src]="company.imageUrl" alt=""/>-->
```

Comment:
Should we provide a dummy image?

Reply:
I dont think so coz that would invite the question why didnt u put actual images ;p